.mergify.yml

```diff
@@ -0,0 +1,69 @@
+pull_request_rules:
+  - name: remove outdated reviews for non trusted authors
```

Comment:
Extremely minor nit: could we name this as "external contributors" or "non-core contributors" or something that sounds less judgmental about the level of trust involved? :-)